<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createFileDialog</class>
 <widget class="QDialog" name="createFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建文件</string>
  </property>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChooseDir">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="BidsManager.qrc">
     <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="fileDir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正在创建一个新的记录文件...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>目录：</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnCancle">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateFile">
      <property name="text">
       <string>创建</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="BidsManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnChooseDir</sender>
   <signal>clicked()</signal>
   <receiver>createFileDialog</receiver>
   <slot>btnChooseDirClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreateFile</sender>
   <signal>clicked()</signal>
   <receiver>createFileDialog</receiver>
   <slot>btnCreateFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancle</sender>
   <signal>clicked()</signal>
   <receiver>createFileDialog</receiver>
   <slot>btnCancleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>createFileDialog</receiver>
   <slot>fileNameInputed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnChooseDirClicked()</slot>
  <slot>btnCreateFileClicked()</slot>
  <slot>btnCancleClicked()</slot>
  <slot>fileNameInputed()</slot>
 </slots>
</ui>
